staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch.go

```diff
@@ -394,7 +574,7 @@ func diffListsOfScalars(original, modified []interface{}, diffOptions DiffOption
 		}
 	}
 
-	return addList, deletionList, nil
+	return addList, uniqifyScalars(deletionList), nil
```

Comment:
Ok, sure, that's a good justification. I'm partial to unix convention :)
The spelling of the verb in English is definitely uniquify but I can see how "uniqify" as a word based on the tool "uniq" makes sense.
No need to follow up on this comment @mengqiy -- I think the discussion is resolved. Thanks for your input @apelisse